src/status_im/contexts/wallet/swap/setup_swap/view.cljs

```diff
@@ -1,13 +1,36 @@
 (ns status-im.contexts.wallet.swap.setup-swap.view
-  (:require [quo.core :as quo]
+  (:require [clojure.string :as string]
+            [native-module.core :as native-module]
+            [quo.core :as quo]
             [react-native.core :as rn]
             [react-native.safe-area :as safe-area]
+            [status-im.common.controlled-input.utils :as controlled-input]
             [status-im.contexts.wallet.common.account-switcher.view :as account-switcher]
             [status-im.contexts.wallet.common.utils :as utils]
             [status-im.contexts.wallet.swap.setup-swap.style :as style]
+            [utils.hex :as hex]
             [utils.i18n :as i18n]
+            [utils.money :as money]
             [utils.navigation :as navigation]
-            [utils.re-frame :as rf]))
+            [utils.number :as number]
+            [utils.re-frame :as rf]
+            [utils.string :as utils.string]))
+
+(def ^:private token-for-fees-symbol "ETH")
```

Comment:
Constant namespace is a better home for these variables.

Reply:
Moved